content/en/blog/_posts/2022-03-31-ready-for-dockershim-removal.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: blog
+title: "Is Your Cluster Ready for v1.24?"
+date: 2022-03-31
+slug: ready-for-dockershim-removal
+---
+
+**Author:** Kat Cosgrove (Pulumi)
+
+
+Way back in December of 2020, Kubernetes announced the [deprecation of Dockershim](/blog/2020/12/02/dont-panic-kubernetes-and-docker/). In Kubernetes, dockershim is a software shim that allows you to use the entire Docker engine as your container runtime within Kubernetes. In the upcoming v1.24 release, we are removing Dockershim - the delay between deprecation and removal in line with the project’s policy of supporting features for at least one year after deprecation. If you are a cluster operator, this guide includes the practical realities of what you need to know going into this release. Also, what you need to do to make sure your cluster doesn’t fall over!
+
```

Comment:
> If you are a cluster operator, this guide includes the practical realities of what you need to know going into this release. 

This _isn't_ the release notes - I'm genuinely worried that someone will think that Kubernetes v1.24 is imminently due and that this is the entire upgrade guide.

Reply:
Reasonable -- I will rephrase this to avoid any confusion